bindings/python/tests/fdbcli_tests.py

```diff
@@ -215,6 +215,25 @@ def kill(logger):
     assert new_generation > old_generation
 
 
+@enable_logging()
+def killall(logger):
```

Comment:
I think we need to call this, right?

Reply:
Oh, nice find. Added it.
I am having a ctest in joshua to test if `new_gen_number == old_geneartion_number` always holds

Reply:
I update it to `new_gen_number == old_geneartion_number + 2`
Tested with 50K runs.